app/routes/_gcn.circulars.$circularId.($version)/route.tsx

```diff
@@ -0,0 +1,165 @@
+}: LoaderFunctionArgs) {
+  if (!feature('CIRCULAR_VERSIONS') && version)
+    throw new Response(null, { status: 404 })
+  invariant(circularId)
+  const result = await get(
+    parseFloat(circularId),
+    version ? parseFloat(version) : undefined
+  )
+
+  return json(result, {
+    headers: {
+      ...publicStaticShortTermCacheControlHeaders,
+      ...getCanonicalUrlHeaders(
+        new URL(
+          `/circulars/${circularId}${version ? `/${version}` : ''}`,
+          origin
+        )
+      ),
+    },
+  })
+}
+
+export const headers: HeadersFunction = ({ loaderHeaders }) =>
+  pickHeaders(loaderHeaders, ['Link'])
+
+export default function () {
+  const { circularId, body, bibcode, version, ...frontMatter } =
+    useLoaderData<typeof loader>()
+  const searchString = useSearchString()
+
+  const result = useRouteLoaderData<typeof parentLoader>(
+    'routes/_gcn.circulars.$circularId'
+  )
+
+  const latest = version && !result?.history.includes(version)
+  const linkString = `/circulars/${circularId}${
+    !latest && version ? `/${version}` : ''
+  }`
+  return (
+    <>
+      <ButtonGroup>
+        <Link to={`/circulars${searchString}`} className="usa-button">
+          <div className="position-relative">
+            <Icon.ArrowBack
+              role="presentation"
+              className="position-absolute top-0 left-0"
+            />
+          </div>
+          &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Back
+        </Link>
+        <ButtonGroup type="segmented">
+          <Link
+            to={`${linkString}.txt`}
+            className="usa-button usa-button--outline"
+            reloadDocument
+          >
+            Text
+          </Link>
+          <Link
+            to={`${linkString}.json`}
+            className="usa-button usa-button--outline"
+            reloadDocument
+          >
+            JSON
+          </Link>
+          {bibcode ? (
+            <Link
+              to={`https://ui.adsabs.harvard.edu/abs/${bibcode}`}
+              className="usa-button usa-button--outline"
+            >
+              Cite (ADS)
+            </Link>
+          ) : (
+            <Button
+              type="button"
+              disabled
+              outline
+              title="The ADS entry for this Circular is not yet available. ADS entries are updated every week on Monday at 08:00 UTC. Please check back later."
+            >
+              Cite (ADS)
+            </Button>
+          )}
+        </ButtonGroup>
+        {useFeature('CIRCULAR_VERSIONS') && result?.userIsModerator && (
+          <Link
+            to={`/circulars/edit/${circularId}`}
+            className="usa-button usa-button--outline"
+          >
+            Edit
+          </Link>
+        )}
+      </ButtonGroup>
+      <h1 className="margin-bottom-0">GCN Circular {circularId} </h1>
+      {useFeature('CIRCULAR_VERSIONS') && (
+        <CircularsHistory circular={circularId} history={result?.history} />
+      )}
```

Comment:
Perhaps we should only show the version info if there is more than one version (history is not empty)?